공유 버튼 클릭시 알람을 띄워줘야 한다 (#123)

happydhKim opened the pull request:
# 공유 버튼 클릭시 알람을 띄워줘야 한다 (#123)

## 해당 이슈 📎

- 공유 버튼 클릭시 알람을 띄워줘야 한다 (#123)

## 변경 사항 🛠

- 공유 버튼을 클릭 시 알람을 띄워준다.
- 알람은 3초후 자동으로 제거된다.

## 테스트 ✨

> 없음

## 리뷰어 참고 사항 🙋‍♀️

> 없음

happydhKim committed:
[ FEATURE ] 채널코드가 복사 되면, 알림 표시가 필요하다. (#123)

Why
- 채널코드가 복사 되면, 알림 표시가 나타나야 한다.

What
- Mertirial UI의 tooltip을 세팅한다.
- hover가 아닌 버튼을 눌렀을 경우 tooltip을 보이게 한다.
- 3초후에 자동으로 사라지게 한다.

happydhKim committed:
[ MERGE ] 최신버전으로 업데이트 (#123)

Why
- hotfix로 의해 업데이트된 내용을 업데이트 해야 한다.

What
- 최신버전으로 업데이트.

devjaewon commented on web/src/components/channel/Slide/SlideInfo/CodeShareButton/CodeShareButton.jsx:
```diff
@@ -1,19 +1,46 @@
-    </SmallButton>
+    <ClickAwayListener onClickAway={handleTooltipClose}>
+      <Tooltip
+        PopperProps={{
```
프로퍼티가 한개인데, 컨벤션 상 한줄로 쓰는 것이 좋을 것 같습니다.

happydhKim replied:
앗 그렇네요 바로 수정하겠습니당🤭

devjaewon reviewed (approved):
굳굳~ 금방 작업하셨네요 👍

always-awake commented on web/src/components/channel/Slide/SlideInfo/CodeShareButton/CodeShareButton.jsx:
```diff
@@ -1,19 +1,46 @@
+import S from './style';
 
 const CodeShareButton = () => {
+  const [open, setOpen] = useState(false);
```
open, setOpen 보다는.. 예를 들면, isToolTipOpen, setIsToolTipOpen 처럼 좀더 구체적이었으면 좋겠습니당 🙆‍♀️

happydhKim replied:
오호 좋네요!! 수정하겠습니당!!

always-awake reviewed (approved):
수고많으셨습니당 짱짱 💃

happydhKim committed:
[ REFACTOR ] 리뷰에서 받은 내용을 수정 (#123)

Why
- 리뷰에서 받은 내용을 수정해야 한다.

What
- 기존 상태 open, setOpen을 toolTipOpen, setToolTipOpen으로 변경.
- 팀 컨벤션 수정 ( 파라미터가 하나 일 시, 한줄로 )
